<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formUser</class>
 <widget class="QDialog" name="formUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabUser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1291</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="perfil">
    <attribute name="title">
     <string>Perfil</string>
    </attribute>
    <widget class="QTextEdit" name="infousuarioA">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>461</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mis Datos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eliminar">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>370</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar Cuenta</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrar">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>370</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Pefil</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Solicitudes</string>
    </attribute>
    <widget class="QLineEdit" name="correoBuscar">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buscar">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="encontrado">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese un Correo:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enviarsol">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Enviar Solicitud</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrarsol">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>30</y>
       <width>171</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Solicitudes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aceptars">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>480</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rechazars">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>480</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechazar</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textsolicitudes">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>120</y>
       <width>361</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>0</y>
       <width>16</width>
       <height>601</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Publicaciones</string>
    </attribute>
    <widget class="QTextEdit" name="textPubli">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>301</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarFe">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por Fecha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inlimit">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>InOrden</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="limite">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>240</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="poslimit">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>PostOrden</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prelimit">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>PreOrden</string>
     </property>
    </widget>
    <widget class="QPushButton" name="verco">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>270</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver Comentarios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="comentarbutton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>270</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Comentar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="horapu">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>350</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>493</x>
       <y>0</y>
       <width>20</width>
       <height>501</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="contenidoP">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="agregarP">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>80</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Publicar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textofecha">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingrese la fecha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fechapu">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>350</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textocom">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>430</y>
       <width>311</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>320</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>320</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hora:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>400</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Comentario:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eliminarpubli">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>270</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Reportes</string>
    </attribute>
    <widget class="QPushButton" name="toppubli">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>201</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Top 3 Publicaciones</string>
     </property>
    </widget>
    <widget class="QPushButton" name="topcomen">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>181</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Top 3 Comentadas</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textotops">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>431</width>
       <height>401</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Heavy</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>542</x>
       <y>70</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Grafico por Fecha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textgrafico">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>70</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingrese una fecha:</string>
     </property>
    </widget>
    <widget class="QLabel" name="graficobst">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>120</y>
       <width>801</width>
       <height>471</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="salir">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
